library/lib_exec_node.ks

```diff
@@ -0,0 +1,43 @@
+@lazyglobal off.
+
+function MaxShipThrust{ 
+	// this function is needed as a workaround
+	// to kOS bug regarding ship thrust.
+	local engs is list().
+	list engines in engs.
+	local mth is 0.
+	for eng in engs{
+		if eng:VISP > 0 {
+			set mth to mth + eng:MAXTHRUST.
+		}
+	}
+	return mth.
+}
+
+function exec_node{
+	parameter nd.
+	
+	local mth is MaxShipThrust().
+	lock burntime to nd:deltav:mag/(mth/mass).
```

Comment:
1. if maximum thrust is 0 (e.g. the player forgot to activate engines) it will crash with cryptic error message